ovn_cluster.sh

```diff
@@ -298,30 +301,50 @@ function wait-containers() {
 # Starts OVN dbs RAFT cluster on ovn-central-1, ovn-central-2 and ovn-central-3
 # containers.
 function start-db-cluster() {
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH} --db-nb-addr=170.168.0.2 --db-nb-create-insecure-remote=yes  \
---db-sb-addr=170.168.0.2 --db-sb-create-insecure-remote=yes --db-nb-cluster-local-addr=170.168.0.2 \
---db-sb-cluster-local-addr=170.168.0.2 start_ovsdb
+    local ssl_certs_path=$1
+    SSL_ARGS=""
+    if [ "$ENABLE_SSL" == "yes" ]; then
+        SSL_ARGS="--ovn-nb-db-ssl-key=${SSL_CERTS_PATH}/ovn-privkey.pem \
+--ovn-nb-db-ssl-cert=${SSL_CERTS_PATH}/ovn-cert.pem \
```

Comment:
nit: can we indent this to be inside the if block?

Reply:
Done